src/lib.rs

```diff
@@ -308,6 +308,19 @@ impl Entry {
     /// Create an entry for the given service and user.
     ///
     /// The default credential builder is used.
+    ///
+    /// # Errors
+    ///
+    /// This function will return an [`Error`] if the `service` or `user` values are invalid.
```

Comment:
Please don't use backticks around links, so [Error] not [`Error`] (see examples at the top of the file).

Reply:
Sure. I was actually just copying the std library format: https://doc.rust-lang.org/stable/src/std/sync/rwlock.rs.html#292-301